configure.ac

```diff
@@ -137,6 +137,13 @@ fi
 AS_IF([test "x$with_Bsymbolic" = "xyes"], [JSON_BSYMBOLIC_LDFLAGS=-Wl[,]-Bsymbolic-functions])
 AC_SUBST(JSON_BSYMBOLIC_LDFLAGS)
 
+# Enable symbol versioning on GNU/Linux
+JSON_SYMVER_LDFLAGS=
+AC_CANONICAL_HOST
+AS_CASE([$host],
+  [*-linux-gnu*],
```

Comment:
please allow any `*gnu*` OS in addition to `linux*` ones, as the toolchain is the same

Reply:
The problem here is that on the CMake side, I'm not sure how to check for glibc, only for a particular kernel. This creates a risk that people start compiling jansson on GNU/kFreeBSD or whatever by using Autotools, and it gets the versioned symbols; and then they move to compiling with CMake on the same platform and it loses them, which is a non-backwards-compatible ABI change.

(Or perhaps you aren't concerned about that, in which case this is fine.)

Reply:
FWIW, in my experience maintaining libraries with autotools + cmake, the output of the two build systems are rarely ABI-compatible with each other.